package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/algolia/autocomplete.js.git"
   },
   "dependencies": {
-    "npm-zepto": "^1.1.7"
+    "zepto": "^1.0.1"
```

Comment:
Yep so the version number actually is not at all related to the real underlying zepto version. But that's the latest: https://libraries.io/npm/zepto. https://github.com/madrobby/zepto/issues/1157